services/src/atdd/src/test/java/org/openkilda/atdd/FlowCrudBasicRunTest.java

```diff
@@ -80,13 +81,17 @@ public void successfulFlowCreation(final String flowId, final String sourceSwitc
         flowPayload = new FlowPayload(FlowUtils.getFlowName(flowId),
                 new FlowEndpointPayload(sourceSwitch, sourcePort, sourceVlan),
                 new FlowEndpointPayload(destinationSwitch, destinationPort, destinationVlan),
-                bandwidth, false, flowId, null, FlowState.UP.getState());
+                bandwidth, false, false, flowId, null, FlowState.UP.getState());
 
         FlowPayload response = FlowUtils.putFlow(flowPayload);
         assertNotNull(response);
-        response.setLastUpdated(null);
 
-        assertEquals(flowPayload, response);
+        System.out.println(String.format(
```

Comment:
It looks like we need to think about logging...

Reply:
We definitely should.

But there is a plan to move out from atdd a lot of test that actually not acceptance tests. So there is a sense to "clean" such issues after planned reorganization.